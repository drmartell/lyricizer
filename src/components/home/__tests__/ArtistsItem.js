import React from 'react';
import { shallow } from 'enzyme';
import ArtistItem from '../ArtistsItem';

describe('ArtistItem component', () => {
  it('should render ArtistItem', () => {
    const wrapper = shallow(<ArtistItem />);
    expect(wrapper).toMatchSnapshot();
  });
});
