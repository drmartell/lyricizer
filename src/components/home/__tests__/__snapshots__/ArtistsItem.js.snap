// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistItem component should render ArtistItem 1`] = `
<section>
  <h2 />
</section>
`;
